htdocs/core/lib/functions.lib.php

```diff
@@ -7920,3 +7921,242 @@ function roundUpToNextMultiple($n, $x = 5)
 {
 	return (ceil($n)%$x === 0) ? ceil($n) : round(($n+$x/2)/$x)*$x;
 }
+
+/**
+ * @param string $label     label of badge no html : use in alt attribute for accessibility
+ * @param string $html      optional : label of badge with html
+ * @param string $type      type of badge : Primary Secondary Success Danger Warning Info Light Dark status0 status1 status2 status3 status4 status5 status6 status7 status8 status9
+ * @param string $mode      default '' , pill, dot
+ * @param string $url       the url for link
+ * @param array $params     various params for future : recommended rather than adding more fuction arguments
+ * @return string   html badge
+ */
+function dol_get_badge($label, $html = '', $type = 'primary', $mode = '', $url = '', $params = array())
```

Comment:
When we introduce new method/function, we must use the camel case syntax, so dolGetBadge

Reply:
ok for me, as you can see i have renamed function to be like others dol_print_* functions